app/controllers/v0/message_drafts_controller.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+module V0
+  class MessageDraftsController < SMController
+    def create
+      params = draft_create_params
+      draft = MessageDraft.new(params)
+      raise Common::Exceptions::ValidationErrors, draft unless draft.valid?
+      response = client.post_create_message_draft(params)
+
+      render json: response,
+             serializer: MessageSerializer,
+             meta:  {}
+    end
+
+    def update
+      params = draft_update_params
+      draft = MessageDraft.new(params)
+      raise Common::Exceptions::ValidationErrors, draft unless draft.valid?
+
+      response = client.post_create_message_draft(params)
+
+      render json: response,
+             serializer: MessageSerializer,
+             meta:  {}
```

Comment:
Looking over the update again - we do not need to use separate parameters for updating and creating a draft. Since our route for update is PUT /message_drafts/:id, requiring the FE to incorporate the message :id in both the payload and url seems redundant. I am taking the liberty of creating one set of strong params for both create and update drafts, and adding the :id parameter from the url to the parameters passed in for update.